conf/st2.conf.sample

```diff
@@ -6,17 +6,21 @@
 enable = True
 
```

Comment:
Here we go - 40018836044f1942208c7bff23678aa170358cd0, ccbfb0e566272c26e069ee584125fab064831725